platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -97,6 +104,20 @@ private void wire() {
         orphanBufferScheduler.getEventOutput().solderTo(inOrderLinkerScheduler.getEventInput());
         inOrderLinkerScheduler.getEventOutput().solderTo(linkedEventIntakeScheduler.getEventInput());
```

Comment:
This pattern makes it look like schedulers are all adhering to a common interface, but every scheduler defines their own get input wire and get output wire method.  The conformance to a pattern is happenstance.   Should there be a common interface or common base class that explains and enforces the similarity?  (Feel free to say no)

Reply:
Let me know if the refactor mitigates your concerns - there wasn't any interface added for this, but things have been reorganized, and naming improved.